### PR TITLE
Prepare repo structure for release

### PR DESCRIPTION
**Pull request by Fryzu**
Refactors repo structure with https://github.com/callstack/react-native-builder-bob template.

### CODE REVIEW

**Review by lukmccall — APPROVED** [review]

File: example/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "dependencies": {
+    "react": "18.1.0",
+    "react-native": "0.70.3",
+    "@react-navigation/bottom-tabs": "^6.3.1",
+    "@react-navigation/native": "^6.0.10",
+    "@react-navigation/native-stack": "^6.6.2",
+    "react-native-safe-area-context": "^4.3.1",
+    "react-native-screens": "^3.13.1"
+  },
```
Comment (aleqsio) [inline example/package.json:20]: Assuming in the future this will have the deps to `react-native-turbo`?

**Review by aleqsio — COMMENTED** [review]
Awesome!

**Review by aleqsio — APPROVED** [review]

**Review by Fryzu — COMMENTED** [review]
👀

File: example/package.json (hunk 1 above)
Comment (Fryzu) [reply example/package.json:20]: Right now any changes to the root folder library js/ts code will be reflected without rebuild. Thats why its configured with 
```js
    'react-native-turbo-webview': {
      root: path.join(__dirname, '..'),
    },
 ```
https://github.com/software-mansion-labs/react-native-turbo-demo/blob/08bfb3606d5937c9296090e6bbe7a3d265928b0e/example/react-native.config.js not with the package.json. Please let me know if I have misunderstood the question.

**Pull request merged**